Makefile

```diff
@@ -83,40 +83,19 @@ clean:
 test:
 	@$(GPGME_ENV) GO111MODULE="on" go test $(BUILDFLAGS) -cover ./...
 
-# This is not run as part of (make all), but Travis CI does run this.
-# Demonstrating a working version of skopeo (possibly with modified SKOPEO_REPO/SKOPEO_BRANCH, e.g.
-#    make test-skopeo SKOPEO_REPO=runcom/skopeo-1 SKOPEO_BRANCH=oci-3 SUDO=sudo
-# ) is a requirement before merging; note that Travis will only test
-# the master branch of the upstream repo.
-test-skopeo:
-	@echo === Testing skopeo build
-	@project_path=$$(pwd) && project_module=$$(GO111MODULE="on" go list .) && export GOPATH=$$(mktemp -d) && \
-		skopeo_path=$${GOPATH}/src/github.com/containers/skopeo && \
-		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
-		cd $${skopeo_path} && \
-		GO111MODULE="on" go mod edit -replace $${project_module}=$${project_path} && \
-		make vendor && \
-		make BUILDTAGS="$(BUILDTAGS)" bin/skopeo test-unit-local && \
-		$(SUDO) make BUILDTAGS="$(BUILDTAGS)" check && \
-		rm -rf $${skopeo_path}
-
 fmt:
-	@gofmt -l -s -w $(SOURCE_DIRS)
+	@go fmt -l -s -w $(SOURCE_DIRS)
 
 validate: lint
 	@GO111MODULE="on" go vet ./...
-	@test -z "$$(gofmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
+	@test -z "$$(go fmt -s -l . | grep -ve '^vendor' | tee /dev/stderr)"
```

Comment:
Is there more context for these changes? AFAICT `go fmt` doesn’t have (and never had?) the `-s -l -w` options, this seems to break local `make`.

Reply:
ugh, darn, I wish there was (in hind-sight).  It's stretching my memory quite a bit but I do remember having problems with running 'gofmt'.  Since those options are invalid, I bet the command is (incorrectly) passing because 'pipefail' defaults to disabled in bash.  In any case, I'll take a look into this and open up a new PR to fix it.  Thanks for tracking it down.

Reply:
For the record, this is being addressed in #1255 .